Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,289 @@
+        )
+    )
+
+label monika_imdrinking:
+    if mas_getEV('monika_imdrinking').shown_count == 0:
+        m 1eua "Hey, [player]..."
+        m 4eub "Remember how I told you earlier to make sure you're staying hydrated?"
+        m 4eua "I just wanted to know if you listened."
+        m 3hua "Are you drinking anything at the moment, [player]?"
+    else:
+        m 3hub "You know, I'm getting a little thirsty."
+        m 3eua "How about you, [player]?"
+        m 1eub "Are you having anything to drink?"
+
+    python:
+        import store.mas_food as mas_food
+
+        #Build the drink list
+        food_menu_items = [
+            (ev.prompt, ev.eventlabel, False, False)
+            for ev_label, ev in mas_food.food_db.iteritems()
+            if 'drink' in ev.category
+        ]
+
+        #Sort the drink list
+        food_menu_items.sort()
+
+        #Create the 'Nothin' option
+        final_item = (mas_food.FOOD_RETURN, False, False, False, 20)
+
+    #Display the scrollable
+    show monika at t21
+    call screen mas_gen_scrollable_menu(food_menu_items, mas_food.FOOD_AREA, mas_food.FOOD_XALIGN, final_item=final_item)
+    show monika at t11
+
+    #Call appropriate label (or exit dlg)
+    if _return:
+        $ pushEvent(_return)
+        $ mas_food_current = _return
+    else:
+        m 1lud "Promise me you'll get a glass of water at least."
+        m 1eud "I just want to make sure you're staying healthy and hydrated, my love."
+        m 1eua "Even if that means waiting for you while you get it."
+    return
+
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_food_database,
+            eventlabel='mas_food_coffee',
+            prompt='Coffee',
+            category=['drink'],
+            unlocked=True
+        ),
+        code="FOO"
+    )
+
+label mas_food_coffee:
+    if mas_getEV(persistent._mas_acs_enable_coffee).shown_count == 0:
+        m 1eua "That sounds wonderful!"
+        m "I really miss having coffee."
+        m 3eub "I would always have a cup before and after school."
+        m 3hub "Maybe even sometimes during class."
+        m 2hub "Who knows, maybe you and I could share a glass, [player]."
+    else:
+        m "That's really cool, [player]!"
+        m "I'm so glad you gave me some earlier."
+        m "Now we can drink some together."
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_food_database,
+            eventlabel='mas_food_water',
+            prompt='Water',
+            category=['drink'],
+            unlocked=True
+        ),
+        code="FOO"
+    )
+
+label mas_food_water:
+    m 1eua "I'm really happy to hear that, [player]!"
+    m 1eub "Water is probably the healthiest thing that you could drink."
+    m 3eua "Seeing how we can only live for a few days without it."
+    m 3eud "Just promise you get plenty to drink each day."
+    m 2lud "I don't want you getting sick because you didn't drink any."
```

Comment:
Again, I personally don't think `lud` is the right expression to convey nervousness or concern. 

Maybe you could add a sweat drop, knit eyebrows and maybe a nervous laughing mouth?  (mouth b)